contracts/validator/contract/XDCValidator.sol

```diff
@@ -1,9 +1,7 @@
-
```

Comment:
1. Are changes to this solidity file aligned with `validator.go` file? In `validator.go`, I didn't see the corresponding changes.
2. For the changes to the smart contract, are they compatible to the smart contract before change? For example, before change, the smart contract stores candidates at an address X. After the change is it still at address X? If not, the smart contract after change cannot read candidates.

Reply:
There are no changes in the smart contract and validator.go

Reply:
![image](https://user-images.githubusercontent.com/10041790/137623363-1b6c916e-76c7-4dd6-a598-70b0b67030c0.png)
that's a massive change, no?